ckanext/switzerland/helpers.py

```diff
@@ -62,24 +62,13 @@ def get_org_count():
 
 
 def get_app_count():
-    result = _call_wp_api('app_statistics')
-    if result is not None:
-        return result['data']['app_count']
-    return 0
-
-
-def _call_wp_api(action):
```

Comment:
Is removing this function needed for the counting of showcases?
Maybe you can remove this function in its own commit that explains, why the function is no longer needed?